auth/token-managers/iam-request-based-token-manager.ts

```diff
@@ -41,11 +41,11 @@ export interface IamRequestOptions extends JwtTokenManagerOptions {
  * class be extended with specific implementations.
  */
 export class IamRequestBasedTokenManager extends JwtTokenManager {
-  private clientId: string;
+  protected clientId: string;
 
-  private clientSecret: string;
+  protected clientSecret: string;
 
-  private scope: string;
+  protected scope: string;
```

Comment:
I adjusted the visibility of these so that I can override them in the IAM Assume token manager (otherwise, they would be added to the request if set).